### DISCUSSION TITLE
여러분!! 개발 과정에서 테스트 돌려보시나요?

### DISCUSSION THREAD

**Discussion by ryan-g00** [prgrms-fe-devcourse/NFE1_2_2_MUX#55] [B. 일반]
저 궁금한건데 API 불러와야 하는 부분은 개발 과정에서 테스트 돌려보시고 PR 올려주시는건가요?
더미 데이터로 하신 분도 계신 것 같고 API 정보 끌어다가 쓰신 분도 계신 것 같은데
만약 더미 데이터 사용하신 분들은 개발 끝나고 한꺼번에 테스트 돌려볼 예정이신건가요?
그러면 유지보수에 시간이 많이 들어가지 않을지.. 여러분의 의견이 궁금합니다..!

@jny4867 @Hyun-bin-Kim @hjlee2778 @seunghyuni

**Comment by hyunbin78** [comment]
나중에 예를 들어, 제꺼 추천포스트 업로드했을때 PostFeed페이지와 UserPosts로 이동되어야 하는데 만약 더미데이터를 생성하면 그때마다 더미데이터를 없애고 바꿔야하는데  부모 컴포넌트 하나 만들어 놓으면 "PostFeed 담당자"나 "UserPosts 담당자"가 개발할 때 "추천 포스트 업로드 담당자"인 제가 붙어서 부모 컴포넌트 여기서 끌어다 써야한다 이런식으로 해야 할 것 같아요!

**Comment by hjlee2778** [comment]
근데 제 생각은, 저희가 어쨌든 프로그래머스 api를 사용하여 유저 데이터를 저장하는데, 더미데이터로 확인을 하는게 아니라 유저 데이터에 저장이 되는지 까지를 해당 기능 구현을 맡은 분이 확인해야 된다고 생각합니다..! 만약 거기서 문제가 발생했을 경우, 빠르게 논의를 통해서 어떤 방식으로 바꿀지 이야기를 나눠봐야 할 것 같아요! 지금 저희가 어쨌든 주 api를 프로그래머스 api로 쓰다보니까 제약점이 좀 많아서 저는 확인을 하고 넘어가야 하는 부분이라고 생각합니다!
